examples/private-vpc-no-zone/main.tf

```diff
@@ -0,0 +1,299 @@
+  distro                          = var.distro
+  hosts_file                      = var.hosts_file
+  tags                            = var.tags
+  associate_public_ip_addr_client = true
+  security_groups_client          = [aws_security_group.client_sec_group.id]
+  subnets                         = {
+    broker = {
+      "us-west-2a" = aws_subnet.server.id
+    }
+    client = {
+      "us-west-2a" = aws_subnet.client.id
+    }
+  }
+  client_count      = 1
+  availability_zone = ["us-west-2a"]
+  egress_rules      = {
+    "SSH" = {
+      description     = "Allow outbound to ssh"
+      from_port       = 22
+      to_port         = 22
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = ["0.0.0.0/0"]
+      self            = null
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "PING" = {
+      description     = "Allow ICMP"
+      from_port       = -1
+      to_port         = -1
+      protocol        = "icmp"
+      enabled         = true
+      cidr_blocks     = ["0.0.0.0/0"]
+      self            = null
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+  }
+  ingress_rules = {
+    "SSH" = {
+      description     = "Allow inbound to ssh"
+      from_port       = 22
+      to_port         = 22
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = [aws_subnet.client.cidr_block]
+      self            = null
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "Kafka" = {
+      description     = "Allow inbound to access the Redpanda Kafka endpoint"
+      from_port       = 9092
+      to_port         = 9092
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "RPC" = {
+      description     = "Allow security-group only to access Redpanda RPC endpoint for intra-cluster communication"
+      from_port       = 33145
+      to_port         = 33145
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "Admin" = {
+      description     = "Allow anywhere inbound to access Redpanda Admin endpoint"
+      from_port       = 9644
+      to_port         = 9644
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "Grafana" = {
+      description     = "Allow anywhere inbound to access grafana end point for monitoring"
+      from_port       = 3000
+      to_port         = 3000
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
```

Comment:
Should this and the promethus SG rule be conditional on state of enable_monitoring?

Reply:
Can't conditionally include defaults